MicroProfile-JWT-Auth/tck-runner/pom.xml

```diff
@@ -190,7 +195,6 @@
                                 <MICRO_JAR>${payara.micro.path}</MICRO_JAR>
                             </environmentVariables>
                             <systemPropertyVariables>
-                                <arquillian.launch>payara-micro-managed</arquillian.launch>
```

Comment:
Why is this being removed? The referenced configuration still exists as far as I can tell

Reply:
Does it? Because when running it with this property it throws an error saying that arquillian profile payara-micro-managed cannot be found, but without the variable it does run payara micro in a managed fashion. Removing this fixes the problem. If it didn't I wouldn't have removed it.

Reply:
https://github.com/payara/MicroProfile-TCK-Runners/blob/master/MicroProfile-JWT-Auth/tck-runner/src/test/resources/arquillian.xml

It exists, if it's not finding it it's going to be some other issue.
It will run without the property, but that configuration in the arquillian.xml won't be referenced (as I understand it anyway)